rollup.config.mjs

```diff
@@ -20,7 +19,7 @@ const postcssMain = postcssConfig({
 
 const RESOLVE_CONFIG = {
 	browser: true,
-	dedupe: ["svelte"],
+	dedupe: ['svelte']
```

Comment:
Should be using double quotes

Reply:
You know how to fix it. :D

Reply:
I kept to your coding style and eslint rules at the end except for conditionals always having curly braces. You can add back in that rule and remove them if you like. Not too many changers there... ~50-60 or so. I'd say run the app and give it a look.

Reply:
Hopefully some of those config file misalignment issues aren't in the codebase, but I indeed was using tabs and ran the eslint file and it verifed. 
 
 
![image](https://user-images.githubusercontent.com/311473/179154840-ee98a084-0841-4d50-8ffe-3238711419c7.png)